test/com/google/enterprise/adaptor/database/ResponseGeneratorTest.java

```diff
@@ -250,11 +255,13 @@ public void testRowToText_specialCharacters() throws Exception {
 
     ResultSet rs = executeQueryAndNext("select * from data");
     resgen.generateResponse(rs, response);
+
     String golden =
         (is(MYSQL) ? "data,data,data\nid,name,quote\n"
-                   :  "DATA,DATA,DATA\nID,NAME,QUOTE\n")
+                   :  (is(SQLSERVER) ? "id,name,quote\n"
```

Comment:
Surely theres a better merge than this, probably restoring what you had starting with golden = "ID...".